api/scripts/prod/generate-knowledge-element-snapshots-for-active-campaigns.js

```diff
@@ -0,0 +1,113 @@
+const bluebird = require('bluebird');
+const { knex } = require('../../db/knex-database-connection');
+const knowledgeElementRepository = require('../../lib/infrastructure/repositories/knowledge-element-repository');
+const knowledgeElementSnapshotRepository = require('../../lib/infrastructure/repositories/knowledge-element-snapshot-repository');
+
+const DEFAULT_MAX_SNAPSHOT_COUNT = 5000;
+const DEFAULT_CONCURRENCY = 3;
+
+function _printUsage() {
+  console.log(`
+  node generate-knowledge-element-snapshots-for-active-campaigns.js [OPTIONS]
+    --maxSnapshotCount <count>  : Nombre de snapshots max. à générer (doit être supérieur à 0, défaut: ${DEFAULT_MAX_SNAPSHOT_COUNT})
+    --concurrency <count>       : Concurrence (doit être supérieur à 0, limite 10, défaut: ${DEFAULT_CONCURRENCY})
+    --help ou -h                : Affiche l'aide`);
+}
+
+function _validateAndNormalizeArgs(commandLineArgs) {
+  if (commandLineArgs.find((commandLineArg) => (commandLineArg === '--help' || commandLineArg === '-h'))) {
+    _printUsage();
+    process.exit(0);
+  }
+  const commandLineArgsLength = commandLineArgs.length;
+  const maxSnapshotCountIndicatorIndex = commandLineArgs.findIndex((commandLineArg) => commandLineArg === '--maxSnapshotCount');
+  let maxSnapshotCount;
+  if (maxSnapshotCountIndicatorIndex === -1 || maxSnapshotCountIndicatorIndex + 1 >= commandLineArgsLength) {
+    maxSnapshotCount = DEFAULT_MAX_SNAPSHOT_COUNT;
```

Comment:
Dans une fonction dédiée, ici on pourrait faire 
```suggestion
    return DEFAULT_MAX_SNAPSHOT_COUNT;
```
cela éviterait le `else` qui suit et ferait gagner en lisibilité.